cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/SampleFlaw.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.monitor.sampling.aggregator;
+
+import com.linkedin.cruisecontrol.monitor.sampling.aggregator.Extrapolation;
+
+
+/**
+ * The sample flaw for a partition that is still treated as valid.
+ */
+public class SampleFlaw {
```

Comment:
I believe we discussed to replace the `flaw` keyword with a word that says `withExtrapolation`?